staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -267,6 +319,44 @@ type PrefixedClaimOrExpression struct {
 	// prefix is prepended to claim's value to prevent clashes with existing names.
 	// +required
 	Prefix *string `json:"prefix"`
+
+	// expression represents the expression which will be evaluated by CEL.
+	// Must produce a string. CEL expressions have access to the contents of the
+	// token claims for claimValidationRules and claimMappings, userInfo for userInfoValidationRules.
```

Comment:
Recommend stating very clearly how the token claims are accessible.  E.g. "token claims are accessable via the X and Y variables and are objects containing the fields defined by ..."